custom/enikshay/views.py

```diff
@@ -108,3 +108,34 @@ def run_task(task_name):
         elif task_requested in ReconciliationTaskForm.permitted_tasks:
             run_task(task_requested)
         return JsonResponse({'message': 'Task queued. You would get an email shortly.'})
+
+
+class DataDumpTaskView(TemplateView):
+    template_name = "enikshay/data_dumps_task.html"
+
+    @method_decorator(require_superuser)
+    def get(self, request, *args, **kwargs):
+        return super(DataDumpTaskView, self).get(request, *args, **kwargs)
+
+    @staticmethod
+    def permitted_tasks():
+        return [choice[0] for choice in DataDumpTaskForm().fields['task'].choices]
```

Comment:
nit: I think you can just call `DataDumpTaskForm.permitted_tasks` directly, since it's the source for `choices`

Reply:
Looks like I did just end up calling that directly haha https://github.com/dimagi/commcare-hq/blob/master/custom/enikshay/views.py#L139